Button/button.css

```diff
@@ -0,0 +1,122 @@
+@import url(../CSS/navbar.css);
+
+@import url(../CSS/slidebar.css);
+
+
+.btn{
+    color: white;
+    min-width: 100px;
+    width: max-content;
+    border-radius: 2px;
+    border: none;
+    margin: 10px;
+    font-weight: 600;
+    padding: 1rem;
+}
+.btn-primary {
+    background-color: var(--solid-primary-color);
+    color: white;
+}
+
+.btn-secondary{
+    background-color: var(--secondary-color) ;
+    color:var(--content-color);
+}
+.btn-secondary:hover{
+    background-color: white;
+    border: 2px solid var(--secondary-color);
+}
+
+.btn-success{
+    background-color: var(--success-color);
+}
+
+.btn-error{
+    background-color: var(--danger-color);
+}
+.btn-warning{
+    background-color: var(--warning-color);
+}
+
+
+
+
+/* Outline Buttons style*/
+.outline-btn-primary{
+    background-color: white;
+    border:1px solid var(--solid-primary-color);
+    color: var(--solid-primary-color);
+    
+}
+
+.outline-btn-secondary{
+    background-color: white ;
+    color:var(--content-color);
+    border:1px solid var(--secondary-color);
+}
+
+.outline-btn-error{
+    background-color: white;
+    border:1px solid var(--danger-color);
+    color: var(--danger-color);
+    
+}
+
+.outline-btn-success{
+    background-color: white;
+    border:1px solid var(--success-color);
+    color: var(--success-color);
+    
+}
+.outline-btn-warning{
+    background-color: white;
+    border:1px solid var(--warning-color);
+    color: var(--warning-color);
+    
+}
+.example-box-btn{
+     
+        border-radius: 4px;
+        background-color: var(--secondary-color);
+        min-height: 80px;
+        width: 95%;
+        /* display: flex; */
+        /* align-items: center;
+        justify-content: center; */
+        /* padding: 1rem;
+        margin-bottom: 1.9rem;
+        padding-left: 2rem;
+        padding-top: 1rem; */
+        padding:1rem 0rem 1.9rem 2rem;
+        
+    
+}
```

Comment:
Hey, I think we shouldn't mix components css with documentation. Maybe you can move it in a different file. Just a suggestion.